Galaxus.Functional.NUnit/Galaxus.Functional.NUnit.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <LangVersion>latest</LangVersion>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>true</IsPackable>
```

Comment:
Packable is enabled by default, no need to turn it on. And again - we don't need the root namespace here!
Also, those empty lines. Whew. :D